code/obj/sandbags.dm

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2025 DisturbHerb
+ *
+ * Originally contributed to the 35 Below Project
+ * Made available under the terms of the CC BY-NC-SA 3.0
+ * Full terms available in the "LICENSE" file or at:
+ * http://creativecommons.org/licenses/by-nc-sa/3.0/us/
+ */
+
+TYPEINFO(/obj/barricade/sandbags)
+	var/list/connects_to_turf = list()
```

Comment:
It's almost tempting to make them connect to walls

Reply:
![image](https://github.com/user-attachments/assets/65d9480c-c7c7-4da3-b4b9-a1bd67017014)

Reply:
yeah, we had this issue with a few similar things we added in 35b; wooden fences, for example, make tons of sense to connect to walls, but wall slanting makes it a bit awkward. maybe could be done with some custom connector sprites, but that's out of scope here.